CMakeLists.txt

```diff
@@ -37,8 +37,8 @@ set( LEAP_UTIL_EXECUTABLE_NAME leap-util )
 
 # http://stackoverflow.com/a/18369825
 if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
-    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 9.0)
-        message(FATAL_ERROR "GCC version must be at least 9.0!")
+    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 10.5)
```

Comment:
I think Debian 11 is still on 10.2. Do we really feel we must have 10.5 over 10.x? A bit of a shame to block out Debian 11 unless we have reason to believe we need to.

Reply:
Probably 10.2 is fine. I'll update the check for 10.2, and try the `ubuntu20.Dockerfile` to regenerate!